### ISSUE TITLE
Link missing for Week 00

### ISSUE THREAD

**Issue by iphyer** [NAU-ACM/ACM-ICPC-Preparation#8]
It looks like the link for Week00's tutorial is missing, this links nothing 'https://github.com/NAU-ACM/ACM-ICPC-Preparation/blob/master/Week1/Sieve-of-Eretosthenes/README.md'

**Comment by fsinan** [comment]
[This is the link](https://github.com/NAU-ACM/ACM-ICPC-Preparation/blob/master/Week01/Sieve-of-Eretosthenes/README.md)

**Comment by BedirT** [comment]
Hi @iphyer ,

The structure goes like this:

- **Tutorial:** The tutorial we made just for this guide, a blog post most likely. If we didn't write one yet, it is an empty link.
- **Other Sources:** Other study sources that we liked and think that are the best out there.
- **Source Codes:** Raw codes for the algorithm given, basically simplest form of it.
- **Questions:** Questions related to the topic given, of course to practice. (And solutions if we solved it yet.)

See, if you write a blog post just for this guide, and let us know, we will check it and if it is good enough we can put your post to **tutorial** section. (If it is not, we will help you fix it and put it after corrections)
